openlogin/homepage/index.test.ts

```diff
@@ -1,21 +1,108 @@
+      {
+        sentTo: testEmail,
+      }
+    );
+
+    expect(email.subject).toBe("Verify your email");
+    const link = findLink(email.html?.links || [], "Confirm my email");
+    expect(link?.text).toBe("Confirm my email");
+    const href = link?.href || "";
+    const context2 = await browser.newContext();
+    const page2 = await context2.newPage();
+    await page2.goto(href);
+    await page2.waitForSelector(
+      "text=Close this and return to your previous window",
+      {
+        timeout: 10000,
+      }
+    );
+    await page2.close();
+
+    await useAutoCancelShareTransfer(page);
+    await useAutoCancel2FASetup(page);
+    await page.waitForURL(`${openloginURL}/wallet/home`, {
+      waitUntil: "load",
+    });
+  });
+  test.afterAll(async ({ browser }) => {
+    browser.close;
+  });
+
+  test(`should display user email on top right`, async ({ context }) => {
+    expect(await page.isVisible(`text=${testEmail}`)).toBeTruthy();
+  });
+  test(`should display welcome message`, async ({ context }) => {
+    expect(await page.isVisible(`text=Welcome, ${testEmail}`)).toBeTruthy();
+  });
+  test(`Clicking language button should display language dropdown `, async ({
+    context,
+  }) => {
+    await page.click('button:has-text("English")');
+    expect(await page.isVisible(`text=German (Deutsch)`)).toBeTruthy();
   });
 
-  test(`title should be "OpenLogin"`, async ({ page }) => {
-    const title = await page.title();
-    expect(title).toBe("OpenLogin");
+  // checks if the support button routes to correct url
+  test(`Clicking 'Support' button should redirect user to correct support page`, async ({}) => {
+    const popupPromise = page.waitForEvent("popup");
+    await page.click(`text=Support`);
+    const popup = await popupPromise;
+    await popup.waitForLoadState();
+    const URL = await popup.url();
+    expect(URL === "https://torus.crisp.help/en/").toBeTruthy();
   });
-  test(`there should be "Get Started" button`, async ({ page }) => {
-    expect(await page.isVisible(`button:has-text("Get Started")`)).toBeTruthy();
+
+  // checks if the learn more button routes to correct url
+  test(`Clicking 'Learn more' button should redirect user to correct docs page`, async ({}) => {
+    const popupPromise = page.waitForEvent("popup");
+    await page.click('a:has-text("Learn more")');
+    const popup = await popupPromise;
+    await popup.waitForLoadState();
+    const URL = await popup.url();
+    expect(
+      URL === "https://docs.tor.us/open-login/what-is-openlogin"
+    ).toBeTruthy();
   });
 
-  test(`click on "Get Started" should show login modal`, async ({ page }) => {
-    await page.click(`button:has-text("Get Started")`);
-    expect(await page.isVisible("text=Welcome onboard")).toBeTruthy();
+  test(`Clicking 'Logout' button should logout user`, async ({}) => {
+    await page.click(`text=Logout`);
+    expect(
+      await page.isVisible(`text=Manage all your web interactions in one place`)
```

Comment:
"Manage all your web interactions in one place" What's the meaning of this text ?

Reply:
Ahh this is a message displayed on the login page. yeah statement is pretty vague.